Source/Core/Core/FifoPlayer/FifoRecorder.h

```diff
@@ -52,7 +53,7 @@ class FifoRecorder
   volatile bool m_WasRecording = false;
   volatile bool m_RequestedRecordingEnd = false;
   volatile s32 m_RecordFramesRemaining = 0;
-  volatile CallbackFunc m_FinishedCb = nullptr;
+  CallbackFunc m_FinishedCb = nullptr;
```

Comment:
You don't need to assign a `nullptr` here, std::function's default constructor is an empty function.